vllm_ascend/models/deepseek_v2.py

```diff
@@ -983,7 +983,3 @@ def forward(
                                    attn_metadata, intermediate_tensors,
```

Comment:
how about rename this file to deepseek instead？

Reply:
hi @wangxiyuan, separate them would be more reasonable and suitable for expansion. We can refer to the VLLM project.

https://github.com/vllm-project/vllm/tree/main/vllm/model_executor/models

Reply:
> https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/deepseek_v2.py#L928
> 
> So looks that there is no clear principle about the model file name and model arch. add a new file with just one line is meanless IMO


> https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/deepseek_v2.py#L928
> 
> So looks that there is no clear principle about the model file name and model arch. add a new file with just one line is meanless IMO

hi @wangxiyan you're right, from a code perspective, having only one line in a file seems meaningless. But the advantage of this is standardization, when adding support for new models, there is no need to modify the original code, but rather to add files, which makes the project more extensible.